### PR TITLE
ssh: add reverse proxy example

### PR DESCRIPTION
**Pull request by yangvw**
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

**Comment by navarroaxel** [comment]
Hi @yangvw, could you please sign the license to pass the `license/cla` check?

File: pages/common/ssh.md [hunk 1]
```diff
@@ -31,6 +27,10 @@
 
 `ssh -J {{username}}@{{jump_host}} {{username}}@{{remote_host}}`
 
+- SSH reverse proxy: Expose a localhost port on the remote server (Set `GatewayPorts yes` in sshd_config to allow internet access):
```
Comment (bl-ue) [inline pages/common/ssh.md:30]: Two things here:
1. Does "localhost port on the remote server" mean a port that is open on the server, localhost referring to the server itself?
2. "Set `GatewayPorts yes` in sshd_config" &mdash; is `sshd_config` on the server or client?

Also, will you please put `sshd_config` in backticks per WIP PR #5128?

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hello @yangvw and thank you for page update!  I've left some comments below for you to review.

File: pages/common/ssh.md [hunk 2]
```diff
@@ -31,6 +27,10 @@
...
+
+`ssh -NTR {{remote_port}}:localhost:{{local_port}} {{remote_host}}`
```
Comment (bergentroll) [inline pages/common/ssh.md:32]: For me this duplicates the `ssh -L {{9999}}:{{example.org}}:{{80}}` example and also a bit inconsistent with it.

File: pages/common/ssh.md [hunk 3]
```diff
@@ -11,13 +11,9 @@
...
 
-- Connect to a remote server using a specific port:
+- Connect to a remote server using a specific port and run a command:
 
-`ssh {{username}}@{{remote_host}} -p {{2222}}`
-
-- Run a command on a remote server:
-
-`ssh {{remote_host}} {{command -with -flags}}`
+`ssh {{username}}@{{remote_host}} -p {{2222}} {{command -with -flags}}`
```
Comment (sbrl) [inline pages/common/ssh.md:16]: I'm not a fan of combining these examples here. It makes the page more dense and difficult to follow for the beginner.

**Comment by bl-ue** [comment]
Closing as the author seems unavailable and the PR appears to be redundant.

**Pull request closed without merging**